internal/ansible/runner/runner.go

```diff
@@ -277,6 +277,11 @@ func (r *runner) Run(ident string, u *unstructured.Unstructured, kubeconfig stri
 		// link the current run to the `latest` directory under artifacts
 		currentRun := filepath.Join(inputDir.Path, "artifacts", ident)
 		latestArtifacts := filepath.Join(inputDir.Path, "artifacts", "latest")
+		// verify if artifacts directory exists and error out if it does not exist
+		if _, err := os.Stat(latestArtifacts); os.IsNotExist(err) {
+			logger.Error(err, "Ansible-runner binary does not exist")
```

Comment:
Hmm, we don't know for sure that the issue is that the binary wasn't found, maybe something like

```suggestion
			logger.Error(err, "Ansible-runner does not appear to have run, the binary may be missing")
```

Reply:
You're already `Lstat`ing the artifacts path below, so you can just check for a non-exist error there and skip symlinking if true.